src/main/java/com/cs525/reversi/models/MatrixRow.java

```diff
@@ -0,0 +1,25 @@
+package com.cs525.reversi.models;
+
+import java.util.List;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+public class MatrixRow {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @ElementCollection
+    private List<String> cells;
```

Comment:
@MustafaMosad List<CellValues> is more appropriate I think... where CellValue can be an enum with
```
public enum CellValue {
    EMPTY{
        @Override
        public String toString() {
            return "e";
        }
    },
    WHITE {
        @Override
        public String toString(){
            return "w";
        }
    },
    BLACK {
        @Override
        public String toString(){
            return "b";
        }
    }
}
```